indico/core/storage/models.py

```diff
@@ -136,6 +136,19 @@ def size(cls):
         )
 
     @declared_attr
+    def md5(cls):
+        """
+        An MD5 hash of the file.
+
+        Automatically assigned when `save()` is called.
+        """
+        return db.Column(
+            db.String,
+            nullable=False,
+            index=True
```

Comment:
Do we ever want to search by hash? Otherwise I wouldn't bother adding an index to the column